source/auth-user-guide.html.md.erb

```diff
@@ -163,51 +158,31 @@ The Digital Prison Services homepage that users in prisons see after clicking on
 
 
 ### Create a service tile:
-##### To identify where you service tile should appear we need to know:
+To identify where you service tile should appear we need to know:
 - Will your service only be accessed by prison staff? If yes, the service tile will be added to the Digital Prison Services homepage.
-- Will your service be accessed by users that sign in with an external Auth account, a Delius account or an @justice.gov.uk account?
-
-##### To create a service tile you need to provide the following information about your service:
-
-- **Name**: (what is shown on the tile e.g Book a secure move, Allocate a POM)
-- **Description**: (short description to appear on the tile e.g Allocate the appropriate offender manager to a prisoner)
-- **Roles**: the role(s) users must have to be able to see the tile. For example: ROLE_ALLOC_MGR
-- **Url / email**: (the url of the service i.e where it should take the user once they have clicked on the tile.. e.g https://allocation-manager-staging.apps.live-1.cloud-platform.service.justice.gov.uk)
-- **Help email**: we need a contact email address for any enquiries about logging in/access from your users... some services set up a generic inbox for this i.e to manage enquiries from users who have been locked out of their account for any reason or login but aren't part of the group they expected to be in etc.
+- Will your service be accessed by any users that sign in with an external Auth account and/or a Delius account?
 
-### Feedback and support
-To add.. 
+To create a service tile you need to provide the following information about your service:
 
-The information teams need to provide to be listed on https://support-dev.hmpps.service.justice.gov.uk/feedback-and-support
-
-### Client credentials
-You will need client credentials to…
+- **Service name**: this will be displayed on the tile e.g Book a secure move
+- **Service description**: a short description that will be displayed on the tile e.g Allocate the appropriate offender manager to a prisoner
+- **Roles**: the role(s) users must have to be able to see the tile
+- **Url / email**: the url of the service i.e where it should take the user once they have clicked on the tile
+- **Help email**: we need a contact email address for any enquiries from your service users about logging in or accessing your service. Some teams set up a generic inbox for this.
```

Comment:
not your change, but we don't use help email anymore - the content for help is now authored in the feedback and support website instead